examples/mesh/mesh_tutorial.py

```diff
@@ -0,0 +1,127 @@
+# bluemira is an integrated inter-disciplinary design tool for future fusion
+# reactors. It incorporates several modules, some of which rely on other
+# codes, to carry out a range of typical conceptual fusion reactor design
+# activities.
+#
+# Copyright (C) 2021 M. Coleman, J. Cook, F. Franza, I.A. Maione, S. McIntosh,
+#                    J. Morris, D. Short
+#
+# bluemira is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# bluemira is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with bluemira; if not, see <https://www.gnu.org/licenses/>.
+
+"""
+Some examples of using bluemira mesh module.
+"""
+
+# %%
```

Comment:
this example fails for me in IPython, maybe we can just make it a normal "script" example for now? You'd need to convert it to ipynb technically too, but I think this could wait. Perhaps I'm the only one with this issue (VS Code):

```python
---------------------------------------------------------------------------
RuntimeError                              Traceback (most recent call last)
Input In [2], in <module>
      1 import os
----> 3 import dolfin
      4 import matplotlib.pyplot as plt
      6 import bluemira.geometry.tools as tools

File ~/miniforge3/envs/bluemira/lib/python3.8/site-packages/dolfin/__init__.py:142, in <module>
    139 from .common.timer import Timer, timed
    140 from .common.plotting import plot
--> 142 from .fem.assembling import (assemble, assemble_system, assemble_multimesh,
    143                              SystemAssembler, assemble_local)
    144 from .fem.form import Form
    145 from .fem.norms import norm, errornorm

File ~/miniforge3/envs/bluemira/lib/python3.8/site-packages/dolfin/fem/assembling.py:34, in <module>
     32 import ufl
     33 import dolfin.cpp as cpp
---> 34 from dolfin.fem.form import Form
     35 from dolfin import MPI
     36 from dolfin.function.multimeshfunction import MultiMeshFunction

File ~/miniforge3/envs/bluemira/lib/python3.8/site-packages/dolfin/fem/form.py:12, in <module>
     10 import ufl
     11 import dolfin.cpp as cpp
---> 12 from dolfin.jit.jit import dolfin_pc, ffc_jit
     15 class Form(cpp.fem.Form):
     16     def __init__(self, form, **kwargs):
     17 
     18         # Check form argument

File ~/miniforge3/envs/bluemira/lib/python3.8/site-packages/dolfin/jit/jit.py:18, in <module>
     16     dolfin_pc = pkgconfig.parse("dolfin")
     17 else:
---> 18     raise RuntimeError("Could not find DOLFIN pkg-config file. Please make sure appropriate paths are set.")
     21 # Copied over from site-packages
     22 def mpi_jit_decorator(local_jit, *args, **kwargs):

RuntimeError: Could not find DOLFIN pkg-config file. Please make sure appropriate paths are set.
```

Reply:
Removed IPython markdown